<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MOPSLinux package builder</string>
  </property>
  <widget class="QComboBox" name="DepSuggestComboBox" >
   <property name="geometry" >
    <rect>
     <x>21</x>
     <y>400</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Dependency</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Suggestion</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>20</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Package name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Build:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>80</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Architecture:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>270</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Changelog:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>380</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Dependencies/suggestions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>140</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>540</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Tags:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DescriptionLanguageComboBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>ru</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>en</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="TagAddButton" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>540</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TagDeleteButton" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>570</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DepDeleteButton" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>430</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>10</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="VersionEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>40</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ArchComboBox" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>70</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>i386</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>i486</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>x86-64</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>ppc</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>noarch</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="BuildEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>100</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ShortDescriptionEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>140</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="DescriptionEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>170</y>
     <width>551</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ChangelogEdit" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>280</y>
     <width>551</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DepAddButton" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>400</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QListWidget" name="TagListWidget" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>570</y>
     <width>521</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TagEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>540</y>
     <width>521</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="DepTableWidget" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>440</y>
     <width>521</width>
     <height>94</height>
    </rect>
   </property>
   <column>
    <property name="text" >
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>condition</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>version</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>type</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="DepVersionEdit" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>400</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DepConditionComboBox" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>400</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>any</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>>=</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>==</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>></string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="DepNameEdit" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>400</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaintainerMailEdit" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>690</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaintainerNameEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>690</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>670</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Maintainer name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>670</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Maintainer e-mail:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>730</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Load data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BuildButton" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>730</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Create XML and build package</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
